src/cdrizzlebox.c

```diff
@@ -80,172 +82,180 @@ compute_bit_value(integer_t uuid) {
     return bv;
 }
 
-/** --------------------------------------------------------------------------------------------------
- * Compute area of box overlap. Calculate the area common to input clockwise polygon x(n), y(n) with
- * square (is, js) to (is+1, js+1). This version is for a quadrilateral. Used by do_square_kernel.
+/** ---------------------------------------------------------------------------
```

Comment:
I recommend comments and parameter information in the format shown in a previous comment.